packages/web/src/scss/components/Dropdown/_Dropdown.scss

```diff
@@ -7,7 +7,6 @@
     z-index: 1;
     display: none;
     width: max-content;
-    max-width: 100%;
```

Comment:
Carefully! 🙂 While removing `max-width` may fix your issue, it will break another use case: long items on narrow viewports: 

![obrazek](https://user-images.githubusercontent.com/5614085/201064810-81ea7f6b-f2d8-494f-9b10-567611fdf749.png)

However, your use case is nothing unusual and we should be ready for that.

Two ideas:

1. Change the parent of your dropdown to hold all buttons in the row and align the dropdown to `bottom-right`:

   ![Snímek obrazovky 2022-11-10 v 11 22 07](https://user-images.githubusercontent.com/5614085/201067694-4524cb38-0a5d-430e-aa8a-8f068e44d9e6.png)

   Would this work for you?

2. Or, refer to viewport instead of parent on this line:

   ```css
   max-width: calc(100vw - 2 * var(--container-padding-inline, 0));
   ```

   It fixes the problem but makes assumptions about its surroundings which doesn't make me happy…

I'll be happy if @crishpeen and @dlouhak find a minute to think about it too.

Reply:
As discussed in person, it turns out there is no proper solution to this because you never know _where_ the dropdown is placed.